rhubarb/lib/opusfile-0.12/COPYING

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 1994-2013 Xiph.Org Foundation and contributors
```

Comment:
Don't forget adding the new libraries to `/LICENSE.md`.

Reply:
Oh, and update the changelog by adding a new `## Unreleased` section at the top.